golem_messages/factories/tasks.py

```diff
@@ -1,18 +1,19 @@
 # pylint: disable=too-few-public-methods,unnecessary-lambda
 import calendar
-from contextlib import suppress
 import datetime
 import time
 import typing
+from contextlib import suppress
```

Comment:
Why? Alphabetically it should be placed where it was before.

Reply:
pycharm formatter sorted this way